doc/terms_x-road_docs.md

```diff
@@ -38,231 +39,176 @@ This document is licensed under the Creative Commons Attribution-ShareAlike 3.0
+**Interaction** – activation procedure of dataservice (single use), bilateral information exchange through dataservice, i.e. request of dataservice by the service client by sending a request, to which the service provider will send a response.
 
 ## 6 Elements of X-Road technology
 
 ### 6.1 Technology in general
 
-**Core technology** (*tuumtarkvara*) – Component of X-Road software, ensuring integrity and verification value of messages between members. Core technology includes central server, configuration proxy and security server.
+**Core technology** – Component of X-Road software, ensuring integrity and verification value of messages between members. Core technology includes central server, configuration proxy and security server.
 
-**Service technology** (*liidestustehnoloogia*) – Component of X-Road software, simplifying or enabling the use of core technology.
+**Service technology** – Component of X-Road software, simplifying or enabling the use of core technology.
 
 ### 6.2 X-Road internal components
 
-**Security server** (*turvaserver*) – standard software solution for using secure data exchange channel of X-Road and ensuring confidentiality, authenticity and integrity of messages/data exchanged on X-Road.
+**Security server** – standard software solution for using secure data exchange channel of X-Road and ensuring confidentiality, authenticity and integrity of messages/data exchanged on X-Road.
 
-> **Turvaserver** – standardiseeritud tarkvara, mis tagab X-tee sõnumite vahendamise kooskõlas X-tee baasprokotollistikuga, tagades andmete konfidentsiaalsuse, autentsuse ja tervikluse.
+**Central components** are central server and configuration proxy.
 
-**Central components** (*kesksed komponendid*) are central server and configuration proxy.
+**Central server** – the component that manages all registrations of an local X-Road instance (security servers, members, subsystems). This global configuration is distributed for all the local and external security servers.  
 
-> **Kesksed komponendid** on keskuse poolt hallatavad tarkvara komponendid, mis tagavad X-tee turvalise andmevahetuskanali loomiseks ja andmeteenuste kasutamiseks vajaliku info kättesaadavuse X-tee liikmete turvaserveritele.
-
-**Central server** (*keskserver*)
-
-**Configuration proxy** (*konfiguratsioonivahendaja*)
+**Configuration proxy** acts as an intermediary between X-Road servers in the matters of global configuration exchange.
 
 ### 6.3 X-Road external components
 
-**Adapterserver** (EE: *liidestuskomponent*; FI: *sovitinpalvelu*)
-
-> **Liidestuskomponent** on tarkvara, mis võimaldab alamsüsteemil vahetada turvaserveriga sõnumeid vastavalt X-tee sõnumiprotokollile.
+**Adapter Service** converts a request or response query (e.g. from REST) to required X-Road SOAP-protocol. 
 
 **Information system** – a system including technological as well as organizational information processing of a member of X-Road
 
```

Comment:
Add following:

The information system (IS) uses and/or provides services via the X-Road.

Reply:
Text 'The information system (IS) uses and/or provides services via the X-Road.' is in incorrect place. It should be added under the section 'Information system', not under the section 'Adapter Service'.